projects/sonar/src/app/record/validation/validation.component.html

```diff
@@ -0,0 +1,93 @@
+<!--
+ SONAR User Interface
+ Copyright (C) 2021 RERO
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, version 3 of the License.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<div class="card mt-3" *ngIf="validation && user && (!isModerator() || !isOwner())">
+  <div class="card-header">
+    <h5 class="m-0" translate>Validation</h5>
+  </div>
+  <div class="card-body">
+    <div class="alert alert-warning m-0"
+      *ngIf="validation.status === validationStatus.IN_PROGRESS || validation.status === validationStatus.ASK_FOR_CHANGES">
+      {{ "The record is currently in status \"\{\{ status \}\}\". It is not
+      visible in public views." | translate: { status: ('validation_status_' + validation.status) | translate } }}
```

Comment:
two `translate` pipe ? The first one seems just a `format-string-like` pipe ?

Reply:
It's because I already have a translation for 'validation_status_xxx', as it is used alone in some places.